components/monitoring/grafana/base/dashboards/image-controller/OWNERS

```diff
@@ -0,0 +1,20 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+# Should be kept in sync with components/monitoring/grafana/base/dashboards/image-controller/OWNERS
+
+approvers:
+- acmiel
+- mmorhun
+- tkdchen
+- rcerven
+- mkosiarc
+- stuartwdouglas
+- susdas
+
+reviewers:
+- acmiel
+- mmorhun
+- tkdchen
+- rcerven
+- mkosiarc
+- stuartwdouglas
+- susdas
```

Comment:
Its my `redhat user id` .. do we need to specify `github user id`?

Reply:
I am not sure

Reply:
github

Reply:
updated with github name